view/dashboard.html

```diff
@@ -4,44 +4,45 @@
         <meta charset="utf-8">
         <title>MusicPlaya</title>
         <link rel="stylesheet" href="https://bootswatch.com/4/darkly/bootstrap.min.css">
-        <link rel="stylesheet" href="dashboard.css">
+        <link rel="stylesheet" href="../styles/dashboard.css">
     </head>
 
     <body>
-
         <div class="header">
             <h1>Welcome to MusicPlaya</h1>
-            <div class="search_type">
-                <h3 class="song_menu">Songs</h3>
-                <h3 class="playlist_menu">Playlists</h3>
+            <div class="dashboard_menu_type">
+                <p class="song_menu"><a href="./dashboard.html">Songs</a></p>
+                <p class="playlist_menu"><a href="./playlist.html">Playlists</a></p>
             </div> 
         </div>
         <div class="music_search">
+            <div class="search_type">
```

Comment:
FrontEnd looks good, just not sure if we're suppose to be implementing the search_type logic to filter searching, as mentioned in comment above.